### ISSUE TITLE
pycapnp coopts SIGUSR1

### ISSUE THREAD

**Issue by kormat** [jparyani/pycapnp#101]
I'm not sure what the mechanism is here, but when i import capnp, it overrides my SIGUSR1 handler silently. This isn't visible from python, at all, which lead to me spending 90 mins figuring out what was happening, while getting increasingly frustrated. I eventually found https://groups.google.com/forum/#!msg/capnproto/Baza1_LKSjo/5bseZRGZAQAJ

Is it possible to make this stop happening?

**Comment by jparyani** [comment]
Are you using pycapnp's RPC functionality? If not, the change in https://github.com/jparyani/pycapnp/pull/96 should fix your problem. That PR has stalled, but I can take it over if there's demand.

**Comment by kormat** [comment]
We're not, no. Having to opt-in for the RPC functionality would solve this nicely.

**Comment by jcpetruzza** [comment]
Is this issue fixed? I still see capnp coopting SIGUSR1 in the latest version (0.5.9). To reproduce:

```python
#!/usr/bin/env python
from __future__ import print_function
import signal
from time import sleep

import capnp
print('USING CAPNP', capnp.version.version)

def handler(signum, frame):
    print('HANDLING', signum)

signal.signal(signal.SIGUSR1, handler)
signal.signal(signal.SIGUSR2, handler)

while True:
    print('RUNNING')
    sleep(10)
```

Only SIGUSR2 triggers the handler unless the `import capnp` is removed

**Comment by jparyani** [comment]
Unfortunately, it wasn't fixed until 4488194f59f8785da02336820cee5b389c552a2b which happened right after the 0.5.9 release. I've pushed v0.5.10 with the fix included.

**Comment by jcpetruzza** [comment]
Excellent, thanks!
